### PR TITLE
Helm chart refactor

### PR DESCRIPTION
**Pull request by CL-SHLOMIKONCHA**

### CODE REVIEW

File: helm/values.yaml [hunk 1]
```diff
@@ -69,11 +22,63 @@ rasterCommon:
         timeoutSec: 5
         retries: 3
       logToConsole: false
+  serviceUrls:
+    jobManager: http://localhost:8084
```
Comment (asafMasa) [inline helm/values.yaml:26]: shouldn't it be empty?

File: helm/values.yaml [hunk 2]
```diff
@@ -122,4 +127,29 @@ route:
...
+      storage: 5Gi
+      hostPath: /layerSources
+    internalPvc:
+      create: false #for local deployment only
+      storage: 4Gi
+    ingestionSourcePvc:
+      create: false #for local deployment only
+      storage: 4Gi
+    accessModes: ReadWriteOnce
+    storageClassName: host-path
```
Comment (asafMasa) [inline helm/values.yaml:155]: end of file

File: helm/values.yaml [hunk 3]
```diff
@@ -122,4 +127,29 @@ route:
...
+      name: local-sources-pv
+      storage: 5Gi
+      hostPath: /layerSources
+    internalPvc:
+      create: false #for local deployment only
+      storage: 4Gi
+    ingestionSourcePvc:
+      create: false #for local deployment only
+      storage: 4Gi
+    accessModes: ReadWriteOnce
```
Comment (asafMasa) [inline helm/values.yaml:154]: why once and not many?

**Review by asafMasa — CHANGES_REQUESTED** [review]
image
revert please

File: helm/values.yaml (hunk 1 above)
Comment (CL-SHLOMIKONCHA) [reply helm/values.yaml:26]: yup, fixed

File: helm/values.yaml (hunk 3 above)
Comment (CL-SHLOMIKONCHA) [reply helm/values.yaml:154]: bug from the past i didnt touch it on this PR but fixed

File: helm/templates/_helpers.tpl [hunk 4]
```diff
@@ -2,8 +2,12 @@
 Expand the name of the chart.
 */}}
 {{- define "gpkg-merger.name" -}}
+{{- if .Values.isExporter }}
```
[the remark this answers is not in the document]
Comment (CL-SHLOMIKONCHA) [reply helm/templates/_helpers.tpl:5]: CLI have nothin with the helm chart deployment it is not running with it

**Review by asafMasa — APPROVED** [review]

**Pull request merged**